pkg/controlplane/instance.go

```diff
@@ -707,10 +666,18 @@ func (m *Instance) InstallAPIs(apiResourceConfigSource serverstorage.APIResource
 			m.GenericAPIServer.AddPostStartHookOrDie(name, hook)
 		}
 
-		apiGroupsInfo = append(apiGroupsInfo, &apiGroupInfo)
+		if len(groupName) == 0 {
```

Comment:
so, now we have `InstallAPIs` that install both legacy and new ... before we had `InstallLegacyAPI` that in addition to install the legacy APIs started the bootstrap controller ... we still keep the order of execution and run the bootstrap controllers before install the APIs

Reply:
yes.